ui/pages/swaps/smart-transaction-status/canceled-icon.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+export default function CanceledIcon() {
```

Comment:
I've added a few icons for this feature directly in the STX Status folder, but I'm not sure if we already have some of them in some reusable folder. I could also move all of them to a reusable folder (which one?), so they could be used by other features as well in the future.

Reply:
I think this is fine for now. There will be a larger reorganization related to icons in the next couple of months.